collections/_article/dev-snapshot-godot-4-6-dev-5.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Dev snapshot: Godot 4.6 dev 5"
+excerpt: TODO
+categories: [pre-release]
+author: TBD
+image: /storage/blog/covers/dev-snapshot-godot-4-6-dev-5.jpg
+image_caption_title: TBD
+image_caption_description: A game by TBD
+date: 2025-12-01 12:00:00
+---
+
+As the days get longer and colder, we get closer to our much anticipated feature freeze. Dev 5 is jam packed with new features and enhancements as contributors try to finish off their big contributors before we hit the feature freeze and Beta 1 shortly after. As a reminder, feature freeze is the period where we stop merging anything except for bug fixes as we shift our focus to polishing the existing code in preparation for release. We plan on entering our feature freeze on December 3 and only releasing one more dev snapshot before entering the Beta phase. 
```

Comment:
Days are getting shorter in the northern hemisphere:

```suggestion
As the days get shorter and colder, we get closer to our much anticipated feature freeze. Dev 5 is jam packed with new features and enhancements as contributors try to finish off their big contributors before we hit the feature freeze and Beta 1 shortly after. As a reminder, feature freeze is the period where we stop merging anything except for bug fixes as we shift our focus to polishing the existing code in preparation for release. We plan on entering our feature freeze on December 3 and only releasing one more dev snapshot before entering the Beta phase. 
```

That said, it may be better to find a formulation that doesn't assume the reader is located in the northern hemisphere 🙂